sale_generator/models/partner.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+#  @author Sébastien BEAU <sebastien.beau@akretion.com>
+#  @author Abdessamad HILALI <abdessamad.hilali@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def close_from_customer_wizard(self):
+        id = self.env.context.get('from_generator_id')
```

Comment:
Please avoid using python's reserved names as variable names.
You can call this `generator_id` for example.